components/camel-google/camel-google-functions/pom.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.camel</groupId>
+    <artifactId>camel-google-parent</artifactId>
+    <version>3.9.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>camel-google-functions</artifactId>
+  <packaging>jar</packaging>
+  <name>Camel :: Google Functions</name>
+  <description>Camel Component for Google Cloud Platform Functions</description>
+
+  <properties>
+    <firstVersion>3.9.0</firstVersion>
+    <google-cloud-functions-version>1.0.8</google-cloud-functions-version>
```

Comment:
This should move to the parent pom

Reply:
Wonder if we can do like some of the other Google components and import the `libraries-bom`? E.g like here:

https://github.com/apache/camel/blob/master/components/camel-google/camel-google-pubsub/pom.xml#L49-L55